package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trainee-ui-app",
-  "version": "0.65.1",
+  "version": "0.65.3",
```

Comment:
this might be a typo but no big deal.

Reply:
I am not sure the order we want to merge it. But @Judge40's PR is having `0.65.2`
https://github.com/Health-Education-England/tis-trainee-ui/pull/860/files

Reply:
Ah OK, I think you may have got there first! 😄